src/language/class-modifiers.md

```diff
@@ -116,14 +116,14 @@ base class MockVehicle          // ERROR: Cannot be implemented
 
 ## `interface`
 
-Use the `interface` modifier to define an interface. Interface classes can be
-implemented outside of the current library, but not extended. This guarantees:
+To define an interface, use the `interface` modifier. Libraries outside of the
+interface's own defining library can implement the interface, but not extend it.
+This guarantees:
 
-- All members of the class must have an implementation.
-- Users of the API are prevented from overriding a subset of methods.
-- A solution to the [fragile base class problem][], or the freedom to change the
-interface's implementation, because there are no inherited member implementations
-outside of the current library whose behavior needs to be preserved. 
+- When an instance method in the class calls another instance method on `this`,
+it will always invoke a known implementation of the method from the same library.
+- Other libraries can't override methods which may then be called by the interface
+class's own methods in unexpected ways. This reduces the [fragile base class problem][].
```

Comment:
**issue:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).
```suggestion
- When a class instance method calls another instance method on `this`,
the first method invokes a known implementation of the method from the same library.
- Other libraries can't override methods that the interface class's own
methods might call later in unexpected ways. This reduces the [fragile base class problem][].
```